<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>546</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <widget class="QTabWidget" name="Tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="OutputTab">
       <attribute name="title">
        <string>TransformPage</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QPushButton" name="playButton">
          <property name="text">
           <string>paly/pause</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSlider" name="horizontalSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>18</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Source</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Output</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="sourceDisplayLabel">
               <property name="text">
                <string>Source</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="outputDisplayLabel">
               <property name="text">
                <string>Output</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="detectStartButton">
          <property name="text">
           <string>Start Detect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Source List</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QTableView" name="listTableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ConfigTab">
       <attribute name="title">
        <string>ConfigPage</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Common Config</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QPushButton" name="inputFileSelectButton">
                 <property name="text">
                  <string>Select Source File/Path</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="outputPathSelectPathButton">
                 <property name="text">
                  <string>Select Output Path</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2" colspan="2">
                <widget class="QLineEdit" name="outputLineEdit"/>
               </item>
               <item row="3" column="0">
                <widget class="QCheckBox" name="alertAfterCompleteCheckBox">
                 <property name="text">
                  <string>Altert After Complete</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2" colspan="2">
                <widget class="QComboBox" name="comboBox">
                 <item>
                  <property name="text">
                   <string>Single File</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Floder</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Model</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0" colspan="2">
                <widget class="QPushButton" name="selectModelButton">
                 <property name="text">
                  <string>Select Model</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLineEdit" name="modelPathLineEdit">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Confidence</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1" colspan="2">
                <widget class="QLineEdit" name="confidenceLineEdit"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
